src/index.ts

```diff
@@ -53,6 +53,20 @@ module.exports = (app) => {
     });
   }
 
+  const closeIssue = async function (issue, app, issueId) {
+    const github = await app.auth(12536268); // Insert Installation ID here
```

Comment:
Use a environment variable for any unique identification as now its a public repo :sweat_smile: , do it for all these variables

Reply:
and any one can trace back commits or event do easy db searchs